packages/suite/src/components/suite/SecurityCheck/DeviceCompromised.tsx

```diff
@@ -11,37 +11,47 @@ import { useDevice, useDispatch } from 'src/hooks/suite';
 import { captureSentryMessage, withSentryScope } from 'src/utils/suite/sentry';
 import { SecurityCheckFail } from 'src/components/suite/SecurityCheck/SecurityCheckFail';
 
+const reportCheckFail = (checkType: 'revision' | 'hash', contextData: any) => {
+    withSentryScope(scope => {
+        scope.setLevel('error');
+        scope.setTag('deviceAuthenticityError', `firmware ${checkType} check failed`);
+        captureSentryMessage(
+            `Firmware ${checkType} check failed! ${JSON.stringify(contextData)}`,
+            scope,
+        );
+    });
+};
```

Comment:
the tags and message for firmware _revision_ stay the same (continuity in sentry)